grails-app/services/org/bbop/apollo/ChadoHandlerService.groovy

```diff
@@ -33,10 +33,12 @@ class ChadoHandlerService {
     def transcriptService
     def cdsService
 
-    private static final SEQUENCE_ONTOLOGY = "sequence"
-    private static final RELATIONSHIP_ONTOLOGY = "relationship"
-    private static final FEATURE_PROPERTY = "feature_property"
-
+    private static final String SEQUENCE_ONTOLOGY = "sequence"
```

Comment:
Yes, I agree. The Chado export is going to be an incremental effort since it will have to be updated each time Apollo changes the way it handles information (Genotype, Phenotype, etc). So I'll be sure to polish it as we move forward.